joark/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/joark/dokument/PdfDokument.kt

```diff
@@ -22,7 +22,7 @@ import java.time.LocalDate
 private const val FORKLARING_ENDRING = "Forklaring for endring"
 class PdfDokument(val dokument: Inntektsmelding) {
 
-    private val pdf = PdfBuilder(bodySize = 20) // Setter skriftstørrelsen på labels og text
+    private val pdf = PdfBuilder(bodySize = 20, innsendt = "Innsendt: ${dokument.tidspunkt.toNorsk()}") // Setter skriftstørrelsen på labels og text
```

Comment:
Har vi ikke holdt IM-spesifikke detaljer utenfor PdfBuilder? Det hadde vært fint å opprettholdt det.

Reply:
Det ville vært bedre, men ville ikke bruke for mye tid på dette og saksbehandlere venter på at vi skal legge til funksjonaliteten.

Reply:
Renamet til topText og gjorde den optional, litt bedre kanskje?

Reply:
Meget 👌